docs/data/toolpad/reference/api/get-context.md

```diff
@@ -33,14 +33,15 @@ a `ServerContext` containing information on the context the backend function was
 
 ### ServerContext
 
-This described a certain context under which a backend function was called.
+This describes a certain context under which a backend function was called.
 
 **Properties**
 
-| Name        | Type                                    | Description                                       |
-| :---------- | :-------------------------------------- | :------------------------------------------------ |
-| `cookies`   | `Record<string, string>`                | A dictionary mapping cookie name to cookie value. |
-| `setCookie` | `(name: string, value: string) => void` | Use to set a cookie `name` with `value`.          |
+| Name        | Type                                                               | Description                                                                         |
+| :---------- | :----------------------------------------------------------------- | :---------------------------------------------------------------------------------- |
+| `cookies`   | `Record<string, string>`                                           | A dictionary mapping cookie name to cookie value.                                   |
+| `setCookie` | `(name: string, value: string) => void`                            | Use to set a cookie `name` with `value`.                                            |
+| `user`      | `{ name: string; email: string; avatar: string; roles: string[] }` | Get current [authenticated](/toolpad/concepts/authentication/) logged-in user data. |
```

Comment:
* Let's add a separate table for this?
* Would it make sense to call this `session` instead of `user`?

Reply:
`session` sounds good, it's what it's called in `@auth/core` too, `user` might be a bit too generic.

Reply:
I've made these changes, ended up having `session` in context which has a `user` property with this data.
Also covered this in one of the authentication tests (since the user needs to be logged-in to see the context session data).